### ISSUE TITLE
Godot 4 update

### ISSUE THREAD

**Issue by ZachAR3** [uheartbeast/walker-level-gen#3]
Hey Ben,

Like with the tilemap to 3D project I have updated the code to work in Godot 4, and was wondering if you would accept a PR or if I'm better off just making a clone of the repo.

**Comment by rein-m** [comment]
@ZachAR3 would you mind putting up a repo of your v4.x walker? I am also working on this walker algorithm, and would be interested in seeing your implementation. thanks!

**Comment by ZachAR3** [comment]
> @ZachAR3 would you mind putting up a repo of your v4.x walker? I am also working on this walker algorithm, and would be interested in seeing your implementation. thanks!

Yeah, it's a bit connected to my game ATM since I didn't end up cleaning it up for a vid or PR since he never responded, but I will see what I can do.

**Comment by ZachAR3** [comment]
> @ZachAR3 would you mind putting up a repo of your v4.x walker? I am also working on this walker algorithm, and would be interested in seeing your implementation. thanks!

@veil00 
The terrain connector isn't working since I don't wanna take the time to learn how it works, but this basic setup is working and I included the equivalent command to the bitmask command if you wanna setup the terrain with it.
https://github.com/ZachAR3/walker-level-gen-godot-4

**Comment by rein-m** [comment]
@ZachAR3 thanks! much appreciated!

**Comment by rein-m** [comment]
@ZachAR3 would the bitmask command need to be working in order to have the player properly collide with the borders of the tiles?

**Comment by ZachAR3** [comment]
@veil00 I don't think so. As far as I know, it was for the auto tile mapping (e.g the sides show a side texture middle: middle texture, corners : corner texture, etc.), the collisions should still work fine if you create an inverted object or set them as walk-able (I don't use 2D often so I'm not too sure how this is implemented.)
